### PR TITLE
fix fish streams

### PR DESCRIPTION
**Pull request by fazibear**
Fix fish commands to stream data to fzf, and don't wait.

### CODE REVIEW

**Comment by junegunn** [comment]
Thanks. It looks like I fixed the issue in https://github.com/junegunn/fzf/commit/305ec3b3ce16141ecac0b8657ad00b371fc7a133, but accidentally reintroduced the problem in 938151a834c1858ffdd6091cbcb42688a3784b6b. So we only need to fix CTRL-T, CTRL-R and ALT-C don't have the problem. Can you to update and force-push the commit?

File: shell/key-bindings.fish [hunk 1]
```diff
@@ -19,14 +19,14 @@ function fzf_key_bindings
       -o -type f -print \
       -o -type d -print \
       -o -type l -print 2> /dev/null | sed 1d | cut -b3-"
-    eval $FZF_CTRL_T_COMMAND | eval (__fzfcmd) -m > $TMPDIR/fzf.result
+    eval "$FZF_CTRL_T_COMMAND |" (__fzfcmd)" -m > $TMPDIR/fzf.result"
```
Comment (junegunn) [inline shell/key-bindings.fish]: Just a minor nitpick, but I'd like the double-quotes around `(__fzfcmd)` to be symmetric.

**Comment by fazibear** [comment]
Double quotes fixed. I changed all 3 function to make all eval consistance.

**Comment by junegunn** [comment]
Thanks, but what I meant is that we should only fix CTRL-T, and not the other ones. Changing the code that doesn't have the problem makes it harder to understand what the problem was and what our workaround was in retrospect. And please squash the commits.

**Comment by fazibear** [comment]
Done.

**Comment by junegunn** [comment]
Thanks you!

**Pull request merged**